docs/running-from-github.md

```diff
@@ -0,0 +1,43 @@
+# Running this test suite from Github Actions
+
+- Go to the [workflow].
+- Click on 'Run Workflow'
+- Fill in the values as needed. (See [options](#options))
+- Click on the second (green) "Run Workflow" button at the bottom of the form
+  
+
+## Options
+
+### `target-idp-env`
+
+Options are `eval` (default) and `prod`. This is the IdP you will be testing.
+
+### `target-idp-host`
+
+*Optional*. The short name of an IdP host you want to route all the test traffic to, 
+e.g., `idp11` or `idpeval01`.
+
```

Comment:
Is there a reference to the doc that contains these names?

Reply:
I'm not sure. If it turns up, we can add it.

Reply:
(but yes, if there is a doc that someone wants to keep up to date, they can link it :) Just not sure it needs to be a priority.)